rfcs/0067-quarterly-releases.md

```diff
@@ -0,0 +1,66 @@
+    * The release manager will audit the licenses of all project dependencies to ensure they are compatible with Apache
+    * The release manager will open a GitHub issue announcing the release branch cut and target date for the release and state that any further inclusions in the release must be manually cherry-picked
+    * The release manager will create a PR targeted to merge into the release branch with the necessary changes to make a release (i.e. changing version numbers)
+    * Contributors that wish changes to be cherry-picked should comment on the announcement issue with the relevant PRs and commits and their reasoning. The release manager has final say on which changes should be included but should aim to be inclusive at this stage
+    * The release manager begins gathering release notes
+
+* Two weeks prior to release
+    * Cherry picks become limited to critical changes only
+    * The release manager begins building binaries and testing them against TVM's test suite
+    * The release manager sends the release notes and plan to a vote thread for approval (feedback thread will be open for one week)
+    * The release manager performs another license audit of all project dependencies to ensure they are compatible with Apache
+
+* One week prior to release
+    * The release manager sends the release and binaries to a vote thread for approval (thread open for one week). If rejected the building and test process must be started anew and repeated until the PMC approves.
+
+* Day of the release
+    * The release manager publishes the relevant binaries
+    * The release manager closes the release issue
+    * The release manager makes a GitHub release and updates an in-repo file `RELEASE.md` on both `main` and the release branch with the release notes
+
+Much of this can be automated via GitHub Actions on the apache/tvm repo. Eventually (though maybe not for the upcoming release) these will handle all the building, testing, and publishing of releases so the job of the release manager will become simpler over time.
+
+The release manager will use the release notes and discussions with developers to determine the next version number. Releases will continue the current versioning scheme of `major.minor.patch`, with a typical release involving a bump of the minor release version. Patch versions will be used for follow up releases onto a quarterly release, but not for the next quarter's release. The release manager will be responsible for maintaining the release until the next release, which mainly entails putting up a patch release if necessary. The release manager may also delegate this responsibility to another party if both agree.
+
+This RFC will not go into effect until we resolve the API around `relay.build` since that may introduce significant churn in the community. See [the discussion](https://discuss.tvm.apache.org/t/pre-rfc-compilation-configuration-representation/11372) for details.
```

Comment:
I don't see quite understand why this has a place in the RFC since the RFC is about release time lines. Equally leaving something so open-ended is a bit unfair to the end users.

 Playing devils advocate, if it takes a year to fix this - what about bug fixes to 0.8 ?

Reply:
I agree that the particular discussion should be left out. We take cautions to evolve the API and the decision of evolution can happen before/after a release as long as it is clearly communicated

Reply:
As the author of the post, I strongly agree we shouldn't block on it given how long it's gone with no sign of resolution. We're currently releasing `0.x` releases, which are typically considered to be unstable, and any versioning scheme we moved to should also allow for multiple breaking changes in a row (such as repeatedly incrementing the major version in [semantic versioning](https://www.semver.org)).

Reply:
agreed since there's not been a lot of movement on it lately, removed